README.md

```diff
@@ -74,8 +78,16 @@ $ wget https://raw.githubusercontent.com/kubevirt/containerized-data-importer/ku
 
 ### Run the CDI Controller
 
-Deploying the CDI controller is straight forward. Choose the namespace where the controller will run and ensure that this namespace has cluster-wide permission to watch all PVCs and pods.
-In this document the _default_ namespace is used, but in a production setup a namespace that is inaccessible to regular users should be used instead. See [Protecting the Golden Image Namespace](#protecting-the-golden-image-namespace) on creating a secure CDI controller namespace.
+Deploying the CDI controller is straight forward. In this document the _default_ namespace is used, but in a production setup a namespace that is inaccessible to regular users should be used instead. See [Protecting the Golden Image Namespace](#protecting-the-golden-image-namespace) on creating a secure CDI controller namespace.
+
+1. Deploy the controller from the release manifest:
+
+```
+$ VERSION=<cdi version>
```

Comment:
is _latest_ supported here?

Reply:
VERSION will default to latest if not supplied.  After this PR is merged, we'll begin automatically releasing :latest images